src/screens/CreatePost/index.js

```diff
@@ -384,6 +356,10 @@ const CreatePost = () => {
     setListTopic(newArr);
     setHashtags(newArr);
     setListTopicChat(newChat);
+
+    if (!newArr.map((i) => i.topic).includes(params?.topic)) {
+      setSelectedTopic(null);
+    }
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of a condition to check if the selected topic is in the list of topics is a good change for data consistency. However, ensure that `params?.topic` and `setSelectedTopic` are correctly defined and used.
```diff
+ if (!newArr.map((i) => i.topic).includes(params?.topic)) {
+   setSelectedTopic(null);
+ }
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->